### PR TITLE
minor: fix for typos in config in xdoc for SuppressionXpathSingleFilter

### PR DESCRIPTION
**Pull request by romani**

### CODE REVIEW

**Comment by romani** [comment]
Github, generate web site

**Review by pbludov — APPROVED** [review]

File: src/xdocs/config_filters.xml [hunk 1]
```diff
@@ -1545,9 +1545,9 @@ public class FileTwo {
         </p>
         <div class="wrapper">
           <pre>
-&lt;module name="SuppressionXpathSingleFilter&gt;
+&lt;module name="SuppressionXpathSingleFilter"&gt;
```
Comment (rnveach) [inline src/xdocs/config_filters.xml:1548]: We have a test that verifies all XML config examples work and are valid. Why did it not catch this?

**Review by rnveach — CHANGES_REQUESTED** [review]

File: src/xdocs/config_filters.xml (hunk 1 above)
Comment (romani) [reply src/xdocs/config_filters.xml:1548]: I surprised we have this validation
but it is not working completely :( - https://github.com/checkstyle/checkstyle/issues/9012

Comment (rnveach) [reply src/xdocs/config_filters.xml:1548]: We have had this validation for long time. I believe it was the first XDoc validation I created.

It validates XDoc, not javadoc from your example.

It is working. There is a secondary issue for why it was not found.

https://github.com/checkstyle/checkstyle/blob/de2f537d42a0165709c40b74a1757822814d22eb/src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java#L362

All XML examples are suppose to be in `source` while this one is in a `div` and `pre`. This is why validation didn't catch it.

We should fix the `source` tag here, and then examine how we can ensure all examples are in `source` and nothing else.

Comment (romani) [reply src/xdocs/config_filters.xml:1548]: @rnveach , I moved your comment to #9012 and provided possible solution , please allow to merge this PR.

**Comment by romani** [comment]
I got private message approval to merge

**Pull request merged**